satyr.spec.in

```diff
@@ -134,6 +136,8 @@ make check|| {
 %{_libdir}/lib*.so.*
 
 %files devel
+# The complex pattern below (instead of simlpy *) excludes Makefile{.am,.in}:
+%doc apidoc/html/*.{html,png,css,js}
```

Comment:
What about:
```
%doc apidoc
%exclude apidoc/html/Makefile*
```
It is longer but maybe better readable?

Reply:
i did it in the same way as abrt does, but this sounds like good idea, updating

Reply:
reverting back to previous , without "exclude", because of rpmbuild compile time complains:
error: File must begin with "/": apidoc/html/Makefile*
http://abrt.brq.redhat.com:9000/job/satyr-pr/43/console